packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -1330,7 +1369,7 @@ class _DropdownMenuState<T extends Object> extends State<DropdownMenu<T>> {
       actions: <Type, Action<Intent>>{
         _ArrowUpIntent: CallbackAction<_ArrowUpIntent>(onInvoke: handleUpKeyInvoke),
         _ArrowDownIntent: CallbackAction<_ArrowDownIntent>(onInvoke: handleDownKeyInvoke),
-        _EnterIntent: CallbackAction<_EnterIntent>(onInvoke: (_) => _handleSubmitted()),
+        _EnterIntent: CallbackAction<_EnterIntent>(onInvoke: handleEnterKey),
```

Comment:
nit: maybe

```suggestion
        _EnterIntent: CallbackAction<_EnterIntent>(onInvoke: handleEnterKeyInvoke),
```

for consistency with the handle methods above.

Reply:
I choosed to renamed the existing callbacks for consistency.